### ISSUE TITLE
Only English language is available in the Settings, with no other language options.

### ISSUE THREAD

**Issue by mingyueshenyuan** [doZennn/steam-rom-manager#674]
Only English language is available in the Settings, with no other language options。
this is the case both on my win11 computer and on steamdeck，maybe I don't know how to operate, I hope it can help me, thank you

![image](https://github.com/SteamGridDB/steam-rom-manager/assets/41833073/2b6a85f4-626e-4b8c-8b5d-dbf2ebb9b105)

**Comment by cbartondock** [comment]
Yeah the languages are disabled right now because so much is changing, they will be re-enabled soon.
